queries/http/highlights.scm

```diff
@@ -0,0 +1,19 @@
+; inherits: json
+
+; Display errors
+(ERROR) @error
+
+; Comments
+(comment) @comment
```

Comment:
you could add this to http/injections.scm as well

https://github.com/nvim-treesitter/nvim-treesitter/blob/e28b65bb52b253d84d8674facb30c10bf3a84119/queries/python/injections.scm#L26

Reply:
just pushed that